starter-project-backend-g31/src/model/model.ts

```diff
@@ -0,0 +1,17 @@
+import mongoose from "mongoose";
+const dataBase = new mongoose.Schema({
+    author : {
+        type : String,
+        required : true
+    },
+    description : {
+        type : String,
+        required : true
+    },
+    date : {
+        type : Date,
+        default : Date.now()
+    }
+});
+
+export const data_base = mongoose.model('data_base', dataBase);
```

Comment:
better to name the model 'Comment' model instead of 'data_base' which is generic